kitchen.dokken.yml

```diff
@@ -35,7 +33,24 @@ platforms:
       intermediate_instructions:
         - RUN /usr/bin/apt-get update
 
-  - name: opensuse-leap
+  - name: ubuntu-20.04
     driver:
-      image: dokken/opensuse-leap
+      image: dokken/ubuntu-20.04
       pid_one_command: /bin/systemd
+      intermediate_instructions:
+        - RUN /usr/bin/apt-get update
+
+  - name: debian-10
+    driver:
+      image: dokken/debian-10
+      pid_one_command: /bin/systemd
+      intermediate_instructions:
+        - RUN /usr/bin/apt-get update
+
+  - name: debian-11
+    driver:
+      image: dokken/debian-11
+      pid_one_command: /bin/systemd
+      intermediate_instructions:
+        - RUN /usr/bin/apt-get update
+...
```

Comment:
typo?

Reply:
I'm missing something obvious here as I can't see it?

Reply:
the three dots at the end

Reply:
Ah I see, i've been following [this](https://yaml.org/spec/1.2.2/#22-structures) which denotes `...` as the end of a document.